database/mysql_anonymiser.go

```diff
@@ -102,21 +106,43 @@ func (a *MySQLAnonymiser) shouldAnonymise(table, column string) string {
 	return viper.GetString(fmt.Sprintf("anonymise.%s.%s", table, column))
 }
 
-// Useful for keeping primary key or any other non-anonymous fields unchanged.
-func (a *MySQLAnonymiser) keepsSeedValueUnchanged(column string, value interface{}, typ interface{}) (*Cell, error) {
+// KeepsSeedValueUnchanged keeps primary key or any other non-anonymous fields unchanged.
+func (a *MySQLAnonymiser) KeepsSeedValueUnchanged(column string, value, typ interface{}) (*Cell, error) {
```

Comment:
Not sure about this. It's bad form to change visibility of methods just for testing purposes. 

Normally, testing your public (or protected, in the case of languages with inheritance chains) API should be sufficient, as that will implicitly cover private methods too.

If you absolutely, positively have to test a private method, you can do so with reflection—but this should be done _sparingly_.

Reply:
Additionally methods should be named in the imperative case, and this and a couple of others in this file aren't.

Reply:
Yes we should not expose functions just for testing purposes. This function don't need to be tested in isolation in fact

Reply:
I was keeping the test functions in a separate package, which explains all the indecent exposing. I've made them part of the same package now. I've also renamed functions in the imperative case and changed a few things here and there to improve readability.